api/infrastructure/handler/user/getseller.go

```diff
@@ -0,0 +1,42 @@
+package user
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/danielgtaylor/huma/v2"
+	"shanepee.com/api/domain"
+	"shanepee.com/api/infrastructure/handler"
+	"shanepee.com/api/service"
+)
+
+type GetSellerByIDInput struct {
+	ID int64 `path:"id"`
+}
+
+type GetSellerByIDOutput struct {
+	Body *domain.User
+}
```

Comment:
```suggestion
type GetSellerByIDOutput struct {
	Body *domain.User
}
```
แยกเป็น query ใหม่ที่มี aggregate ของ review เลย